src/platform/plugins/shared/data/common/search/search_source/search_source.ts

```diff
@@ -688,12 +689,6 @@ export class SearchSource {
         return addToBody(key, sort);
       case 'pit':
         return addToRoot(key, val);
-      case 'projectRouting':
```

Comment:
Does removing this mean consumers won't be able to override the project routing config?